experiments/integrated/fluxnet/fluxnet_simulation.jl

```diff
@@ -1,12 +1,11 @@
-t0 = Float64(120 * 3600 * 24) # start mid year
+t0 = FT(120 * 3600 * 24)# start mid year
 N_spinup_days = 10
 N_days = N_spinup_days + 30
-dt = Float64(120)
-tf = Float64(t0 + 3600 * 24 * N_days)
-
+dt = FT(120)
+tf = t0 + FT(3600 * 24 * N_days)
 # Number of timesteps between saving output
 n = 15
-t_spinup = Float64(t0 + N_spinup_days * 3600 * 24)
+t_spinup = t0 + FT(N_spinup_days * 3600 * 24)
```

Comment:
I think Julia put `Float64` here to ensure that the time type is Float64 and not inferred from somewhere else when FT = Float32

Reply:
Ah ok. I actually started this branch before the Float32/Float64 compatibility PR so I need to double check and make sure I followed all of the convention Julia laid out for ensuring Float32 compatibility.